banman/store_test.go

```diff
@@ -123,4 +124,10 @@ func TestBanStore(t *testing.T) {
 	// We'll query for second IP network again as it should now be unknown
 	// to the BanStore. We should expect not to find anything regarding it.
 	checkBanStore(ipNet2, false, 0, 0)
+
+	// Test UnbanIPNet.
+	require.NoError(t, banStore.UnbanIPNet(ipNet1))
```

Comment:
it was unbanned above, should ban it again and then check that `UnbanIPNet` unbans it

Reply:
I do not think that it is up to an hour, here?

Reply:
I just mean that we should all `BanIPNet` again before we call `UnbanIPNet` again. I guess I don't understand the question?

Reply:
I mean, it hould still be banned because it is not up to one hour:
https://github.com/lightninglabs/neutrino/blob/eb5942f8b86afe0a51026bfa3bc218fac554e9f5/banman/store_test.go#L114

Reply:
ah I see, the below `checkBanStore` then needs to be `checkBanStore(ipNet1, false, 0, 0)`

Reply:
Thanks